src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,23 @@
+package duke;
+
+import duke.parser.Parser;
+
+/**
+ * Entry point of Duke Chatbot.
+ * Initializes the application and starts the interaction with the user.
+ *
+ * @author Won Ye Ji
+ */
+public class Duke {
+
+    /**
+     * Runs the program until termination.
+     *
+     * @param args arguments supplied by the user at program launch.
+     */
+    public static void main(String[] args) {
+        Parser parser = new Parser();
+        parser.initialiseDuke();
+        parser.runDuke();
+    }
+}
```

Comment:
I think this file is missing a newline at the end of the file? Noticed it for a few other files too.